src/scripts/upload.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "${AVIATOR_API_TOKEN}" ]; then
+    echo "AVIATOR_API_TOKEN is required."
+    exit 1
+fi
+
+if [ -z "${FILE_PATH}" ]; then
+    echo "FILE_PATH is required."
+    exit 1
+fi
+
+if ! which curl > /dev/null; then
+    echo "curl is required to use this command"
+    exit 1
+fi
+
+URL="https://upload.mergequeue.com/api/test-report-uploader"
```

Comment:
please change this to the prod url:
```suggestion
URL="https://upload.aviator.co/api/test-report-uploader"
```

Is there a way to make this a hidden configuration?

Reply:
not sure if we can make it hidden ... can look into it